lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -314,16 +325,18 @@ describe('CategoryRenderer', () => {
     });
   });
 
-  describe('clumping passed/failed/manual', () => {
+  describe('clumping passed/failed/warning/manual', () => {
     it('separates audits in the DOM', () => {
       const category = sampleResults.reportCategories.find(c => c.id === 'pwa');
       const elem = renderer.render(category, sampleResults.categoryGroups);
       const passedAudits = elem.querySelectorAll('.lh-clump--passed .lh-audit');
       const failedAudits = elem.querySelectorAll('.lh-clump--failed .lh-audit');
+      const warningAudits = elem.querySelectorAll('.lh-clump--warning .lh-audit');
       const manualAudits = elem.querySelectorAll('.lh-clump--manual .lh-audit');
 
       assert.equal(passedAudits.length, 4);
       assert.equal(failedAudits.length, 8);
+      assert.equal(warningAudits.length, 0);
```

Comment:
this test doesn't end up testing much :) WDYT about adding a warning to one of the passed audits?

Reply:
There are no passing audits with warnings in the sample json, and I'm not sure how to update the source appropriately. fine to just manually set some warnings in the test case?

Reply:
> fine to just manually set some warnings in the test case

perfect